File: libs/api/src/main/java/com/example/api/graphapi/utils/FbConsts.java
package com.example.api.graphapi.utils;

import java.util.Arrays;
import java.util.Collections;
import java.util.List;

public interface FbConsts {
	List<String> FACEBOOK_USER_FIELDS = Collections.unmodifiableList(
		Arrays.asList(
			Fields.ID,
			Fields.NAME,
			Fields.FIRST_NAME,
			Fields.EMAIL,
			Fields.GENDER,
			Fields.BIRTHDAY,
			Fields.PICTURE_LARGE,
			Fields.HOMETOWN,
			Fields.ALBUMS
		)
	);
}

File: libs/api/src/main/kotlin/com/example/api/graphapi/utils/FbConsts.kt
package com.example.api.graphapi.utils

import java.util.*

interface FbConsts {
  companion object {
    val FACEBOOK_USER_FIELDS: MutableList<String> = Collections.unmodifiableList(
      Arrays.asList(
        Fields.ID,
        Fields.NAME,
        Fields.FIRST_NAME,
        Fields.EMAIL,
        Fields.GENDER,
        Fields.BIRTHDAY,
        Fields.PICTURE_LARGE,
        Fields.HOMETOWN,
        Fields.ALBUMS
      )
    )
  }
}
